config/storage/middleware/stores/patching.json

```diff
@@ -14,11 +14,11 @@
             {
               "comment": "Makes sure PATCH operations on containers target the metadata.",
               "@type": "ContainerPatcher",
-              "patcher": { "@type": "SparqlUpdatePatcher" }
+              "patcher": { "@id": "urn:solid-server:default:PatchHandler_Type" }
```

Comment:
I find the `_Type` suffix weird. Why is it not just `urn:solid-server:default:PatchHandlers`?

Reply:
I find that to be a bit too generic. Could go for `urn:solid-server:default:PatchHandler_RDF` since this is the handler that is wrapped by other patch handlers that convert to quads first.

Reply:
Perfect.